tests/vmi_multus_test.go

```diff
@@ -893,7 +936,7 @@ func configInterface(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAdd
 		&expect.BSnd{S: "echo $?\n"},
 		&expect.BExp{R: "0"},
 	}, 15)
-	Expect(err).ToNot(HaveOccurred(), "Failed to configure address %s for interface %s on VMI %s", interfaceAddress, interfaceName, vmi.Name)
+	Expect(err).ToNot(HaveOccurred(), "Failed to configure address %s for interface %s on VMI %s with error %v", interfaceAddress, interfaceName, vmi.Name, err)
```

Comment:
The is not need to add the error to the message test, it will be printed anyway to the log (please verify it, I'm not 100% sure about it).

Reply:
you right. it will print "expect [the error content including the message] to not have occurred"

Reply:
I see you kept the error print, which is redundant.